spec/memoize_query_spec.rb

```diff
@@ -612,4 +537,364 @@ def expect_to_eq_with_or_without_redis
       expect_to_eq_with_or_without_redis { relation_with_and_or_clause.reload }
     end
   end
+
+  context 'when the queries have compare operators (unbound)' do
+    RSpec.shared_examples 'does not memoize queries with only compare operators' do
```

Comment:
Good point. I refactored the shared examples a bit, but did not end up doing `relations_with_only_one_comparator.each do |relation|` (passing the relation) because of some implementation details for `let`. but I passed a list of `relations` as a let variable.